packages/browser-destinations/src/destinations/heap/__tests__/index.test.ts

```diff
@@ -0,0 +1,22 @@
+import { Analytics, Context } from '@segment/analytics-next'
```

Comment:
We can't run unit tests because `@segment/analytics-next` is a private segment repo. I'm not sure if this unit test works but we will find out when we submit the PR to segment and the CI runs.